### PR TITLE
Deprecate std/sums

### PR DESCRIPTION
**Pull request by planetis-m**
Supersedes #18438 . Refs https://github.com/nim-lang/packages/pull/1963

### CODE REVIEW

File: lib/std/sums.nim [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ## Accurate summation functions.
 
+{.deprecated: "use a nimble package instead.".}
```
Comment (konsumlamm) [inline lib/std/sums.nim]: Perhaps link the package (or just the GitHub repo) or at least mention its name (`sums`).

Comment (planetis-m) [inline lib/std/sums.nim]:
```suggestion
{.deprecated: "use the nimble package `sums` instead.".}
```

**Comment by konsumlamm** [comment]
Any update on this?

**Comment by konsumlamm** [comment]
@Araq can this be done for 2.0?

File: changelog.md [hunk 2]
```diff
@@ -11,6 +11,10 @@
   or define your own `Math.trunc` polyfill using the [`emit` pragma](https://nim-lang.org/docs/manual.html#implementation-specific-pragmas-emit-pragma). Nim uses
   `Math.trunc` for the division and modulo operators for integers.
 
+- Deprecated `std/sums`.
+
+- `cuchar` now aliases `uint8` instead of `char`.
```
Comment (ringabout) [inline changelog.md]:
```suggestion
```

**Comment by ringabout** [comment]
ready for review

**Review by ringabout — APPROVED** [review]

**Pull request merged**